distribution/src/config/jvm.options

```diff
@@ -45,6 +45,14 @@
 # 10-:-XX:+UseG1GC
 # 10-:-XX:InitiatingHeapOccupancyPercent=75
 
+## DNS cache policy
+# cache ttl for positive DNS lookups noting that this overrides the JDK security
```

Comment:
Probably worth mentioning that these values are in seconds in the comments (to avoid the step where users "forget" to check the meaning on the other side).

Reply:
Good call.

Reply:
I pushed 9c38f870310590990b84ea167b7dbf32304dbb30.